prody/proteins/ciffile.py

```diff
@@ -432,7 +436,12 @@ def _parseMMCIFLines(atomgroup, lines, model, chain, subset,
 
         alt = line.split()[fields['label_alt_id']]
         if alt not in which_altlocs and which_altlocs != 'all':
-            continue
+            try:
+                alt = ascii_uppercase[int(alt)]
```

Comment:
Does this work? Wouldn't it make the altloc 'B' when the default is 'A'?

Reply:
Oh, right, true. I need to subtract 1.